Exclude BSD from Linux Clang module support, as in Chromium ports.

Index: codex-rs/modcargo-crates/v8-150.4.0/build/config/clang/clang.gni
--- codex-rs/modcargo-crates/v8-150.4.0/build/config/clang/clang.gni.orig
+++ codex-rs/modcargo-crates/v8-150.4.0/build/config/clang/clang.gni
@@ -40,6 +40,7 @@
   "x86",
 ]
 clang_modules_platform_supported =
+    !is_bsd &&
     # s390x should work, but IBM doesn't want it enabled for that cpu arch.
     (is_linux && current_cpu != "s390x") ||
     (is_android &&
